src/java/main/org/apache/zookeeper/server/persistence/FileTxnLog.java

```diff
@@ -127,14 +127,11 @@
 
         Long logSize = Long.getLong(LOG_SIZE_LIMIT, -1);
         if (logSize > 0) {
+            LOG.info("{} = {}", LOG_SIZE_LIMIT, logSize);
+
```

Comment:
{} value is {} Bytes

Reply:
I renamed the property as per the other comment, I think now the log is readable as is. Please let me know if you think otherwise.